packages/meteor/debug.js

```diff
@@ -1,67 +1,76 @@
-var suppress = 0;
+let suppress = 0;
 
-// replacement for console.log. This is a temporary API. We should
-// provide a real logging API soon (possibly just a polyfill for
-// console?)
-//
-// NOTE: this is used on the server to print the warning about
-// having autopublish enabled when you probably meant to turn it
-// off. it's not really the proper use of something called
-// _debug. the intent is for this message to go to the terminal and
-// be very visible. if you change _debug to go someplace else, etc,
-// please fix the autopublish code to do something reasonable.
-//
-Meteor._debug = function (/* arguments */) {
+/**
+ * Replacement for console.log. This is a temporary API. We should
+ * provide a real logging API soon (possibly just a polyfill for
+ * console?)
+ *
+ * NOTE: this is used on the server to print the warning about
+ * having autopublish enabled when you probably meant to turn it
+ * off. it's not really the proper use of something called
+ * _debug. the intent is for this message to go to the terminal and
+ * be very visible. if you change _debug to go someplace else, etc,
+ * please fix the autopublish code to do something reasonable.
+ */
+Meteor._debug = function(...args) {
```

Comment:
Please keep the space after `function` for all anonymous functions.

Reply:
And also - since the name of this PR includes "modernize", you could consider replacing your anonymous function declarations with arrow functions. In this case specifically it would make sense since you're using spread syntax to capture all function arguments in `args`, but since you're using a function declaration, you're just masking the fact that an `arguments` object is still available (arrow functions don't have `arguments`).